content/blogposts/2022/release-4.1.md

```diff
@@ -0,0 +1,63 @@
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.1/sourcegraph-4-1.png
+changelogItems:
+  - description: Added repository sync counters to the code host details page to give visibility into external service sync progress.
+    url: https://github.com/sourcegraph/sourcegraph/pull/43039
+    category: Repositories
+  - description: "Added a new button in the repository settings, under \"Mirroring,\" to delete a repository from disk and reclone it. This prevents the need to manually delete failed repositories from the Git server."
+    url: https://github.com/sourcegraph/sourcegraph/pull/42177
+    category: Repositories
+  - description: "GraphQL request logs are now compliant with the audit logging format. The old GraphQl logging based on `LOG_ALL_GRAPHQL_REQUESTS` env var is now deprecated and scheduled for removal."
+    url: https://github.com/sourcegraph/sourcegraph/pull/42550
+    category: Admin
+  - description: "Git server access logs are now compliant with the audit logging format. This introduces a breaking change: The 'actor' field is now nested under the 'audit' field."
+    url: https://github.com/sourcegraph/sourcegraph/pull/41865
+    category: Admin
+  - description: Security events are now included in the audit log by default to provide more visibility to security teams.
+    url: https://github.com/sourcegraph/sourcegraph/pull/42653
+    category: Admin
+  - description: "Security events in the audit log can now optionally exclude internal actor traffic to reduce noise. This traffic is excluded by default, but can be enabled with the `log.auditLog.backgroundJobs` setting."
+    url: https://github.com/sourcegraph/sourcegraph/pull/42946
+    category: Admin
+  - description: Fixed a bug with GitHub code hosts that caused archived repos to be incorrectly returned when using the "public" repositoryQuery keyword.
+    url: https://github.com/sourcegraph/sourcegraph/pull/41461
+    category: Admin
+---
+
+<Badge link="/batch-changes" text="Batch Changes" color="blue" size="small" />
+
+#### Server-side Batch Changes now support file mounts and organization namespaces
+
+We're iterating on [server-side Batch Changes](https://about.sourcegraph.com/blog/release/4.0#high-leverage-ways-to-improve-your-entire-codebase) as we prepare to move it from beta to GA. In 4.1, we're adding the last features to bring server-side Batch Changes up to feature parity with batch changes that are run locally.
+- You can now mount files on batch change steps containers using `steps.mount` when running server-side. This is useful when your batch change needs to run long scripts or binaries that change frequently: file mounts allow you to iterate on those scripts and binaries without needing to bake them into the container and rebuild the container every time they change. You can read more about this feature in [release post 3.41](https://about.sourcegraph.com/blog/release/3.41).
```

Comment:
```suggestion
- When running server-side Batch Changes, you can now mount files on batch change steps containers using `steps.mount`. This is useful when your batch change needs to run long scripts or binaries that change frequently because file mounts allow you to iterate on those scripts and binaries without needing to bake them into the container and rebuild the container every time they change. You can read more about this feature in the [3.41 release post](https://about.sourcegraph.com/blog/release/3.41).
```